<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgExportParams</class>
 <widget class="QDialog" name="DlgExportParams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>I want to....</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>512</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="m_bAll">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>166</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Export whole  subtree </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_bOnlySelected">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>166</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Export only selected node</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_bCancel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>166</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="m_ExportPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_bChangePath">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>166</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>166</width>
     <height>27</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>166</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>Choose directory...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
